pkg/exchange/coinbase/api/v1/get_order_trades_request.go

```diff
@@ -0,0 +1,73 @@
+package coinbase
+
+import (
+	"time"
+
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+	"github.com/c9s/bbgo/pkg/types"
+	"github.com/c9s/requestgen"
+)
+
+type Liquidity string
+
+const (
+	LiquidityMaker Liquidity = "M"
+	LiquidityTaker Liquidity = "T"
+	LiquidityOther Liquidity = "O"
+)
+
+type SideType string
+
+const (
+	SideTypeBuy  SideType = "buy"
+	SideTypeSell SideType = "sell"
+)
+
+func (s *SideType) GlobalSideType() types.SideType {
+	switch *s {
+	case SideTypeBuy:
+		return types.SideTypeBuy
+	case SideTypeSell:
+		return types.SideTypeSell
+	}
+	return types.SideTypeNone
+}
+
+type Trade struct {
+	TradeID         int              `json:"trade_id"`
+	ProductID       string           `json:"product_id"`
+	OrderID         string           `json:"order_id"`
```

Comment:
is it integer type?

Reply:
According to the schema and example response listed in the following link:
https://docs.cdp.coinbase.com/exchange/reference/exchangerestapi_getfills

`TradeID` is of type int.
`ProductID` and `OrderID` are `string`s.